shop/flyway.properties

```diff
@@ -1,5 +1,5 @@
 soring.flyway.user=
```

Comment:
I think you have a typo here. It should be spring instead of soring :D

Reply:
Fixed